src/main/webapp/scripts/forum-script.js

```diff
@@ -1,15 +1,24 @@
 import {forumTemplate} from './forum-template.js';
 
-$( document ).ready( getFilters );
-$( document ).ready( getForum );
+$( document ).ready( function() {
+  getForum();
+  getFilters();
+  $('#search-button').click(search);
+  $('#search-button').keyup(function(event) {
+    if (event.keyCode === 13) {
+      event.preventDefault();
+      search();
+    }
+  });
+});
 
 /**
  *  Populates forum-placeholder with forum data
  */
 function getForum() {
   const id = -1;
   const placeholder = $('#forum-placeholder');
-  expandForum(placeholder, id);
+  expandForum(placeholder, id, null);
```

Comment:
Either add a comment for null (and other literals like string and int):

expandForum(placeholder, id, /* search = */ null);

Or better, add a function expandForumWithSearch(placeholder, id, search), then delegate to that from expandForum(placeholder, id).

Reply:
I created a expandForumWithSearch function, which expandForum uses.  That is a such cleaner code thank you.